src/plugins/custom_integrations/public/components/replacement_card/replacement_card.component.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+/** @jsx jsx */
+
+import { css, jsx } from '@emotion/react';
+
+import {
+  htmlIdGenerator,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiText,
+  EuiAccordion,
+  EuiLink,
+  useEuiTheme,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { CustomIntegration } from '../../../common';
+
+export interface Props {
+  replacements: Array<Pick<CustomIntegration, 'id' | 'uiInternalPath' | 'title'>>;
+}
+
+const idGenerator = htmlIdGenerator('replacementCard');
+const alsoAvailable = i18n.translate('customIntegrations.components.replacementAccordionLabel', {
+  defaultMessage: 'Also available in Beats',
+});
+
+const link = (
+  <EuiLink href="#">
+    <FormattedMessage
+      id="customIntegrations.components.replacementAccordion.comparisonPageLinkLabel"
+      defaultMessage="comparison page"
+    />
+  </EuiLink>
+);
+
+/**
+ * A pure component, an accordion panel which can display information about replacements for a given EPR module.
+ */
+export const ReplacementCard = ({ replacements }: Props) => {
+  const { euiTheme } = useEuiTheme();
+
+  if (replacements.length === 0) {
+    return null;
+  }
+
+  const buttons = replacements.map((replacement) => (
+    <EuiFlexItem grow={false}>
+      <span>
+        <EuiButton
+          key={replacement.id}
+          href={replacement.uiInternalPath}
```

Comment:
The base-path needs to be prepended here. e.g. https://github.com/elastic/kibana/blob/be1ee57a03ed52a25a95da4ffdd5ac644a3f2b6c/x-pack/plugins/fleet/public/hooks/use_link.ts#L26